docs/collectors/PhpFpmCollector.md

```diff
@@ -25,6 +34,9 @@ pm.status_path = /fpm-status
 
 Setting | Default | Description | Type
 --------|---------|-------------|-----
+uri | fpm-status | Path part of the URL, with or without the leading / | string
```

Comment:
It comes from the k/v in `get_default_config_help` in [each collector](https://github.com/python-diamond/Diamond/blob/master/src/collectors/phpfpm/phpfpm.py#L33).  However, it looks like this was never filled in for the collector so this table only includes the [default ones](https://github.com/python-diamond/Diamond/blob/master/src/diamond/collector.py#L244)

Can you add the config options to that method? You can reference this [ntp](https://github.com/python-diamond/Diamond/blob/master/src/collectors/ntp/ntp.py#L34) collector if you need something to look at for it